homeassistant/components/camera/bomradarloop.py

```diff
@@ -0,0 +1,381 @@
+"""
+Provie animtaed GIF loops of BOM radar imagery.
```

Comment:
Spelling errors

Reply:
Thanks. Fixed, along with some other copy-editing in comments.